.travis.yml

```diff
@@ -28,7 +28,7 @@ jobs:
             continue
         fi
         echo "Building image for Dockerfile '$file'..."
-        docker build $(dirname "$file")
+        travis_wait 30 docker build $(dirname "$file")
```

Comment:
We can add the travis_wait option, but I don't understand why it takes more than 10 minutes to build.

Reply:
I will merge it since it's urgent, but @AngryMaciek can you please open an issue about the `travis_wait` flag?

Reply:
There is nothing to make an issue about, really.
I install a lot of packages in this environment with `conda` and so the build is stuck for more than 10 minutes at a `Solving environment: ...working... ` stage while Travis does not receive any more logs. This was a problem for him because of its default configuration, it expects things constantly being printed out:
```
No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.
```

I see that the build on `master` failed since I forgot to at the `travis_wait` in one more place it should have been added: building for remote:
```
docker build -t "$remote" $(dirname $file)
```

I will push a fix branch in a few minutes.